website/src/pages/Blog/Tag/styles.ts

```diff
@@ -0,0 +1,113 @@
+import { makeStyles, Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    position: "relative",
+    backgroundImage: "url(/images/png/faq_background.png)",
+    textAlign: "center",
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center top",
+    backgroundSize: "cover",
+    padding: theme.spacing(16, 5, 5),
+    [theme.breakpoints.down("xs")]: {
+      padding: theme.spacing(5, 1),
+      marginTop: theme.spacing(10),
+      backgroundImage: "url(/images/png/faq_background_mobile.png)",
+      backgroundRepeat: "no-repeat",
+      backgroundSize: "cover",
+    },
+  },
+  mainText: {
+    display: "flex",
+    flexWrap: "wrap",
+    textAlign: "center",
+    maxWidth: '670px',
+    margin: 'auto',
+    justifyContent: "center",
+    alignItems: "center",
+    fontSize: '2.625rem',
+    [theme.breakpoints.down("sm")]: {
+      fontSize: "1.5rem",
+    },
+  },
+  sectionDiv: {
+    padding: theme.spacing(3, 0),
+    width: '75%',
+    margin: 'auto',
+    [theme.breakpoints.down("md")]: {
+      padding: theme.spacing(3, 0),
+      width: '80%'
+    },
+    [theme.breakpoints.down("xs")]: {
+      padding: theme.spacing(3, 0),
+      width: '85%'
+    }
+  },
+  blogTitle: {
+    fontSize: '2.625rem',
+    fontWeight: 700,
+    textAlign: 'center',
+    [theme.breakpoints.down('xs')]: {
+      fontSize: '1.5rem',
+    },
+  },
+  blogsWrapper: {
+    justifyContent: 'space-between',
+    [theme.breakpoints.down("sm")]: {
+      justifyContent: 'center',
+    },
+  },
+  cardSize: {
+    maxWidth: "480px !important",
+    paddingTop: theme.spacing(4),
```

Comment:
There is some change in css (ref: https://github.com/openebs/website/pull/210)

Reply:
This will auto added after rebase :)

Reply:
ref: https://github.com/openebs/website/pull/210/files#diff-63af27bc462eb8b5d718572232baac7a03137a42bd14704b72430a21f09c2ecaR110